Composer/packages/client/src/pages/design/new-dialog-modal.tsx

```diff
@@ -19,12 +20,7 @@ const NewDialogModal: React.FC<NewDialogModalProps> = props => {
 
   return (
     <DialogWrapper isOpen={isOpen} onDismiss={onDismiss} {...DialogCreationCopy.DEFINE_CONVERSATION_OBJECTIVE}>
-      <DefineConversation
-        onSubmit={onSubmit}
-        onDismiss={onDismiss}
-        onGetErrorMessage={onGetErrorMessage}
-        enableLocationBrowse={false}
-      />
+      <DefineDialog onSubmit={onSubmit} onDismiss={onDismiss} onGetErrorMessage={onGetErrorMessage} />
```

Comment:
Maybe it's not necessary to split this into two files, given the naming of "defineDialog.tsx" and "new-dialog-modal.tsx" are not clear and not consistenst. 

I think a simple "createDialog.tsx" would work